lib/pf/api.pm

```diff
@@ -1156,6 +1149,35 @@ sub radius_rest_switch_authorize :Public :RestPath(/radius/rest/switch/authorize
 
     return \%mapped_return;
 }
+
+sub handle_accounting_metadata : Public {
+    my ($class, %RAD_REQUEST) = @_;
+    my $logger = pf::log::get_logger();
+    $logger->debug("Entering handling of accounting metadata");
+    my $client = pf::client::getClient();
+    
+    my $mac = pf::util::clean_mac($RAD_REQUEST{'Calling-Station-Id'});
+    if ($RAD_REQUEST{'Acct-Status-Type'} eq 'Start') {
+        #
+        # Updating location log in on initial ('Start') accounting run.
+        #
+        $logger->info("Updating locationlog from accounting request");
+        $client->notify("radius_update_locationlog", %RAD_REQUEST);
```

Comment:
We must return the return the value of radius_update_locationlog because the radius server will answer $RADIUS::RLM_MODULE_FAIL